### PR TITLE
Add correct label to exclude

### LINKED ISSUE #445: Do not add "project" issues to the Backlog
## Problem

Currently, all new issues are added to the [Backlog on the Openverse GitHub board](https://github.com/orgs/WordPress/projects/3#column-14087268).  We have a special board for the Projects (issues with the "project" label), so it's better not to add them to the Backlog.

## Description

Update https://github.com/WordPress/openverse/blob/cc93d225869932fd683a33014b96da8eb831ff6e/.github/workflows/new_issues.yml#L22 with the correct label `🧭 project: thread`

### PR DESCRIPTION
**Pull request by ngken0995**
## Fixes

Fixes #445  by @obulat 

## Description

Replace `project` with ` 🧭 project: thread, 🧭 project: proposal, 🧭 project: implementation plan` because there is a special board for the Projects and not in the backlog.

## Checklist

- [X] My pull request has a descriptive title (not a vague title like`Update index.md`).
- [X] My pull request targets the _default_ branch of the repository (`main`) or a parent feature branch.
- [X] My commit messages follow [best practices][best_practices].
- [X] My code follows the established code style of the repository.
- [N/A] I added or updated tests for the changes I made (if applicable).
- [N/A] I added or updated documentation (if applicable).
- [ ] I tried running the project locally and verified that there are no visible errors.
- [N/S] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Review by stacimc — APPROVED** [review]
LGTM! I'm not _100%_ sold on excluding `project: proposal` and `project: implementation plan` here as well 🤔 But I'm not opposed. I'll defer to other reviewers on that :)

File: .github/workflows/new_issues.yml [hunk 1]
```diff
@@ -19,7 +19,7 @@ jobs:
           project-url: https://github.com/orgs/WordPress/projects/75
           github-token: ${{ secrets.ACCESS_TOKEN }}
           # Exclude the issues with the following labels
-          labeled: project
+          labeled: "🧭 project: thread, 🧭 project: proposal, 🧭 project: implementation plan"
```
Comment (krysal) [inline .github/workflows/new_issues.yml:22]:
```suggestion
          labeled: "🧭 project: thread"
```

Comment (krysal) [reply .github/workflows/new_issues.yml:22]: The proposals and implementations plans are issues that require PRs and reviews, so they shouldn't be ignored.

**Review by krysal — CHANGES_REQUESTED** [review]
Thank you for you help, @ngken0995. Let's keep this as the issue says and limit the exclusion to project threads.

**Review by obulat — APPROVED** [review]
LGTM!

**Pull request merged**